src/commands/search/kitsu-manga.ts

```diff
@@ -0,0 +1,163 @@
+import type { Kitsu } from '#lib/apis/kitsu/Kitsu';
+import { fetchKitsuApi } from '#lib/apis/kitsu/kitsu-constants';
+import { LanguageKeys } from '#lib/i18n/LanguageKeys';
+import { RedisKeys } from '#lib/redis-cache/RedisCacheClient';
+import { apply } from '#lib/utilities/add-builder-localizations';
+import { bold, EmbedBuilder, hideLinkEmbed, hyperlink, time, TimestampStyles } from '@discordjs/builders';
+import { cutText, isNullishOrEmpty } from '@sapphire/utilities';
+import { AutocompleteInteractionArguments, Command, RegisterCommand } from '@skyra/http-framework';
+import { getSupportedUserLanguageT, resolveUserKey } from '@skyra/http-framework-i18n';
+import { APIApplicationCommandOptionChoice, MessageFlags } from 'discord-api-types/v10';
+
+@RegisterCommand((builder) =>
+	apply(builder, LanguageKeys.Commands.Kitsu.Manga.RootName, LanguageKeys.Commands.Kitsu.Manga.RootDescription) //
+		.addStringOption((option) =>
+			apply(option, LanguageKeys.Common.OptionsManga) //
+				.setRequired(true)
+				.setAutocomplete(true)
+		)
+)
+export class UserCommand extends Command {
+	public override async autocompleteRun(_: Command.AutocompleteInteraction, options: AutocompleteInteractionArguments<Options>) {
+		if (options.focused !== 'manga' || isNullishOrEmpty(options.manga)) {
+			return this.autocompleteNoResults();
+		}
+
+		const result = await fetchKitsuApi('manga', options.manga);
+
+		return result.match({
```

Comment:
It's doing an immediate action of the result from a callback, I normally don't put a blank line between those:
```suggestion
		const result = await fetchKitsuApi('manga', options.manga);
		return result.match({
```